packages/core/src/types/IContourSet.ts

```diff
@@ -0,0 +1,52 @@
+import { ContourData, IContour, Point3 } from './';
+
+export interface IContourSet {
+  readonly id: string;
+  readonly sizeInBytes: number;
+  contours: IContour[];
+  _createEachContour(data: ContourData[]): void;
+  _getSizeInBytes(): number;
+  getSizeInBytes(): number;
+  getColor(): any;
+  /**
+   * This function returns the contours of the image
+   * @returns The contours of the image.
+   */
+  getContours(): IContour[];
+  /**
+   * It returns an array of all the points in the glyph
+   * @returns An array of points.
+   */
+  getFlatPointsArray(): Point3[];
+  /**
+   * This function returns the number of contours in the current shape.
+   * @returns The number of contours in the glyph.
+   */
+  getNumberOfContours(): number;
+  /**
+   * It loops through each contour in the `contours` array, and adds the number of
+   * points in each contour to the `numberOfPoints` variable
+   * @returns The number of points in the contours.
+   */
+  getTotalNumberOfPoints(): number;
+  /**
+   * It returns an array of the number of points in each contour.
+   * @returns An array of numbers.
+   */
+  getNumberOfPointsArray(): number[];
+  /**
+   * It returns the points in a contour.
+   * @param contourIndex - The index of the contour you want to get the
+   * points from.
+   * @returns An array of Point3 objects.
+   */
+  getPointsInAContour(contourIndex: number): Point3[];
```

Comment:
Perhaps one of `getPointsInContour` or `getPointsInTheContour`. Having the 'A' in there is a bit awkward.